pkg/minishift/network/settings.go

```diff
@@ -60,14 +87,127 @@ func printNetworkSettings(networkSettings NetworkSettings) {
 func fillNetworkSettingsScript(networkSettings NetworkSettings) string {
 	result := &bytes.Buffer{}
 
-	tmpl, err := template.New("networkScript").Parse(configureNetworkScriptTemplate)
-	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Error creating network script template: %s", err.Error()))
+	tmpl := template.New("networkScript")
+
+	if networkSettings.Disabled {
+		tmpl.Parse(configureNetworkScriptDisabledAddressTemplate)
+	} else if networkSettings.UseDHCP {
+		tmpl.Parse(configureNetworkScriptDynamicAddressTemplate)
+	} else {
+		tmpl.Parse(configureNetworkScriptStaticAddressTemplate)
 	}
-	err = tmpl.Execute(result, networkSettings)
+
+	err := tmpl.Execute(result, networkSettings)
 	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Error executing network script template:: %s", err.Error()))
+		atexit.ExitWithMessage(1, fmt.Sprintf("Error executing network script template: %s", err.Error()))
 	}
 
 	return result.String()
 }
+
+func determineDefaultGateway(ipaddress string) string {
+	ip := net.ParseIP(ipaddress)
+	ip = ip.To4()
+	//ip = ip.Mask(ip.DefaultMask())
+	ip[3] = 1
+	return ip.String()
+}
+
+func executeCommandOrExit(driver drivers.Driver, command string, errorMessage string) string {
+	result, err := drivers.RunSSHCommandFromDriver(driver, command)
+
+	if err != nil {
+		atexit.ExitWithMessage(1, fmt.Sprintf("%s: %s", errorMessage, err.Error()))
+	}
+	return result
+}
+
+func HasNameserversConfigured(driver drivers.Driver) bool {
```

Comment:
This function seems to be only for Linux and macOS. We should add godoc saying it is only for Linux and macOS.

Reply:
NVM, this is only for the VM isn't it?